statsmodels/iolib/summary.py

```diff
@@ -307,16 +312,16 @@ def summary_top(results, title=None, gleft=None, gright=None, yname=None, xname=
           ('Time:', lambda: time_of_day),
           ('Number of Obs:', lambda: [results.nobs]),
           #('No. of Observations:', lambda: ["%#6d" % results.nobs]),
-          ('No. Observations:', lambda: ["%#6d" % results.nobs]),
+          ('No. Observations:', lambda: [num_to_str(results.nobs)]),
           #('Df model:', lambda: [results.df_model]),
-          ('Df Model:', lambda: ["%#6d" % results.df_model]),
+          ('Df Model:', lambda: [num_to_str(results.df_model)]),
           #TODO: check when we have non-integer df
-          ('Df Residuals:', lambda: ["%#6d" % results.df_resid]),
-          #('Df resid:', lambda: [results.df_resid]),
-          #('df resid:', lambda: [results.df_resid]), #check capitalization
-          ('Log-Likelihood:', lambda: ["%#8.5g" % results.llf]) #doesn't exist for RLM - exception
-          #('Method:', lambda: [???]), #no default for this
-          ])
+          ('Df Residuals:', lambda: [num_to_str(results.df_resid)]),
+          # ('Df resid:', lambda: [results.df_resid]),
+          # ('df resid:', lambda: [results.df_resid]), #check capitalization
```

Comment:
Commented out versions necessary?

Reply:
Commented out versions necessary?

Reply:
I don't know. postponed and new open issue for reviewing GenericLikelihoodModel.
this PR makes the minimal changes to fix the bug to get it to work.